spec.md

```diff
@@ -4358,6 +4359,9 @@ property_primop_2expr = property_primop_2expr_keyword ,
 (* Tokens: Annotations *)
 annotations = "%" , "[" , json_array , "]" ;
 
+(* Enable Layer *)
+enablelayer = "enablelayer" , id , { "." , id };
```

Comment:
Could we introduce a new production rule for colorings?

`coloring := id , { "." , id } ;`

Also, to be consistent with the other rules, let's put a space before the semicolon.

Reply:
Oops dropped the space when reworking, good catch ty!